### PR TITLE
Issue #13345: Enable examples tests for MissingCtorCheck

### PR DESCRIPTION
**Pull request by AmitKumarDeoghoria**
### Title: Enable Example Tests for MissingCtorCheck

**Related Issue: #13345** 

**Summary**
This pull request aims to enable example tests for the **MissingCtorCheck** in the project. The implementation focuses on ensuring that the covariant equals checks are correctly validated through comprehensive test cases, improving overall code reliability and quality.

**Key Changes Made**

- Conversion of txt file to Java File
- Enable the test cases by adding expected violations in the expected array for all example tests
- Made corresponding changes in the xml and xml.template file

### CODE REVIEW

**Comment by AmitKumarDeoghoria** [comment]
GitHub, generate website

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingctor/Example1.java [hunk 1]
```diff
@@ -22,7 +23,7 @@ class ExampleDefaultCtor { // OK
 class InvalidExample { // violation, class must have a constructor.
   public void test() {}
 }
-abstract class AbstractExample { // OK
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingctor/Example1.java:25]: Please update violation line above to have violation message `// violation 'Class should define a constructor'`

**Review by romani — CHANGES_REQUESTED** [review]
The only item:

**Comment by AmitKumarDeoghoria** [comment]
GitHub, generate website

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingctor/Example1.java (hunk 1 above)
Comment (AmitKumarDeoghoria) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingctor/Example1.java:25]: Done.

**Review by romani — APPROVED** [review]
ok to merge.

thanks a lot !

**Pull request merged**